tools/code_format/format_python_tools.py

```diff
@@ -24,7 +24,8 @@ def collect_files():
         dirnames[:] = [d for d in dirnames if d not in EXCLUDE_LIST]
         for filename in fnmatch.filter(filenames, '*.py'):
             if not filename.endswith('_pb2.py') and not filename.endswith('_pb2_grpc.py'):
-                matches.append(os.path.join(root, filename))
+                if "test" not in root:
```

Comment:
we should remove this and use the newer class

this way doesnt respect `.yapfignore`